ocs_ci/deployment/networkpolicy.py

```
@@ -0,0 +1,49 @@
+    """
+    if not config.DEPLOYMENT.get("disable_networkpolicy"):
+        logger.debug(
+            "disable_networkpolicy is not set or False, skipping "
+            "allow-all NetworkPolicy override."
+        )
+        return
+
+    namespace = config.ENV_DATA.get(
+        "cluster_namespace", constants.OPENSHIFT_STORAGE_NAMESPACE
+    )
+    logger.info(f"Applying allow-all NetworkPolicy override to namespace '{namespace}'")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)